.scripts/utils.py

```diff
@@ -355,7 +355,7 @@ def validate_banner(path: str, server_name: str) -> list[str]:
     # Incorrect aspect ratio
     if aspect_ratio != 7.8:
```

Comment:
I thought we agreed on requiring server banners to be `468x60` since that seems like the standard size?

Even if we're not doing this `35/5` is 7 `39/5` is the right ratio